xml/installation-installation-rhel-install_rhel_compute_node.xml

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entities.ent"> %entities;
+]>
+<section xml:id="install_rhel_compute_node"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>Using &rhla; as a &compnode;</title>
+ <para>
+  This section outlines how to install a &rhla; &compnode; as a member of a
+  a new or existing cloud created with &productname;.
+ </para>
+ <section xml:id="sec.install_rhel_compute_node.provision-deployer">
+  <title>Setting Up a Deployer for &rhla;</title>
+  <para>
+   Ensure that your environment includes a Subscription Management Tool
+   (SMT) server, as described in <xref linkend="app.deploy.smt_lcm"/>.
+   The following steps presume that the SMT server is your deployer,
+   and will configure an RPM repository for use by &rhla; clients.
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Enable SMT mirroring of the external CentOS 7.4 repository to distribute
+     RHEL-compatible packages to nodes in the cloud.
+    </para>
+<screen>sudo smt-setup-custom-repos --name CentOS --description "CentOS 7.4" --productid 100682 \
+  --exturl https://download.opensuse.org/repositories/systemsmanagement:/Ardana:/8:/CentOS/CentOS_7.3
+sudo smt-repos -e CentOS
+sudo smt-sync
+sudo smt-mirror</screen>
+   </step>
+   <step>
+    <para>
+     CentOS RPM packages will now be stored on the SMT server in
+     <filename>/srv/www/htdocs/repo/RPMMD/CentOS</filename>. Publish them
+     to other nodes with the following command:
+    </para>
+<screen>sudo ln -s /srv/www/htdocs/repo/RPMMD/CentOS /opt/ardana_packager/ardana/rhel7/yum/centos</screen>
+   </step>
+   <step>
+    <para>
+     Add this new repo as a Yum package source to cloud nodes by populating
+     <filename>/etc/yum.repos.d/ardana-centos.repo</filename> in each
+     &rhla; system with the following contents:
+    </para>
```

Comment:
we should a note here  like "If using a remote SMT, follow the same steps for this repo as you are doing for the other repos noting the correct target path on the deployer. Please see Chapter 5 on the install guide for details ....